packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -0,0 +1,1389 @@
+| `emailTemplateTouchPointVariant`      | Enum     | Variant for Email templates touch point. Accepted values: `OKTA_DEFAULT`, `FULL_THEME`.                                    | `OKTA_DEFAULT`    |
+
+#### Theme Touch Point Variant Definitions
+
+##### Sign In Page Touch Point Variant:
+
+> **Note:** For non `OKTA_DEFAULT` variant, `primaryColorHex` will be used for buttons background color and `primaryColorContrastHex` will be used to identify shade for button text
+
+| Enum Value                      | Description                                                                                          |
+| ------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| `OKTA_DEFAULT`                  | Use Okta default Logo with no background image along with Okta default colors in the sign in page.   |
+| `BACKGROUND_SECONDARY_COLOR`    | Use Logo from Theme with `secondaryColorHex` as background color for the sign in page.               |
+| `BACKGROUND_IMAGE`              | Use Logo and background image from Theme.                                                            |
+
+##### End User Dashboard Touch Point Variant:
+
+| Enum Value                      | Description                                                                                                             |
+| ------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `OKTA_DEFAULT`                  | Use Okta default Logo with white background color for logo and side nav bar background color.                           |
+| `WHITE_LOGO_BACKGROUND`         | Use Logo from Theme with white background color for logo and use `primaryColorHex` for side nav bar background color.   |
+| `FULL_THEME`                    | Use Logo from Theme and `primaryColorHex` for logo and side nav bar background color                                    |
+
+##### Error Page Touch Point Variant:
+
+> **Note:** For non `OKTA_DEFAULT` variant, `primaryColorHex` will be used for buttons background color and `primaryColorContrastHex` will be used to identify shade for button text
+
+| Enum Value                      | Description                                                                                          |
+| ------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| `OKTA_DEFAULT`                  | Use Okta default Logo with no background image along with Okta default colors in the error page.     |
+| `BACKGROUND_SECONDARY_COLOR`    | Use Logo from Theme with `secondaryColorHex` as background color for the error page.                 |
+| `BACKGROUND_IMAGE`              | Use Logo and background image from Theme.                                                            |
```

Comment:
@justinabrokwah-okta   please verify the touch point variant applicability

Reply:
Looks good for error pages!

One thing, I think i wouldn't say `Okta default colours in the error page`, because the text and icon colours within the error container don't change with ThemeBuilder and that may confuse users. Maybe something like `Use Okta default logo along with Okta default background colour`

Reply:
thanks for verifying. updated the content